spec/views/versions/_close_ui.html.erb_spec.rb

```diff
@@ -3,11 +3,12 @@
 require 'rails_helper'
 
 RSpec.describe 'versions/_close_ui.html.erb' do
-  let(:object) { double('object', pid: 'druid:abc123') }
+  before do
+    @cocina = instance_double(Cocina::Models::DRO, externalIdentifier: 'druid:abc123')
```

Comment:
I'm curious why we need an ivar assignment here over, say, an `assign()` statement.

Reply:
`assign()` is deprecated and using ivars is the preferred way.

Reply:
Huh, OK! Good to know, thanks.